website/docs/reference/sdks/index.md

```diff
@@ -119,6 +119,7 @@ Here's some of the fantastic work our community has done to make Unleash work in
 - PHP - Symfony ([Stogon/unleash-bundle](https://github.com/Stogon/unleash-bundle))
 - React Native / Expo ([nunogois/proxy-client-react-native](https://github.com/nunogois/proxy-client-react-native))
 - Solid ([nunogois/proxy-client-solid](https://github.com/nunogois/proxy-client-solid))
+- Haskell ([finn-no/unleash-client-haskell](https://github.com/finn-no/unleash-client-haskell))
```

Comment:
We try to keep it a alphabetical sorted list for neutrality. Mind moving it to correct location?

Reply:
Sure, done.